test/e2e-test/containerized_workload_test.go

```diff
@@ -268,7 +268,17 @@ var _ = Describe("ContainerizedWorkload", func() {
 			},
 			time.Second*60, time.Second*5).Should(BeEquivalentTo(replica))
 		Expect(*deploy.Spec.Replicas).Should(BeEquivalentTo(replica))
-
+		By("Verify pod annotations should not contain kubectl.kubernetes.io/last-applied-configuration")
+		Eventually(
+			func() bool {
+				k8sClient.Get(ctx, objectKey, deploy)
+				annotations := deploy.Spec.Template.Annotations
+				if _, ok := annotations["kubectl.kubernetes.io/last-applied-configuration"]; ok {
```

Comment:
the workload created by go-client will not have this annotations. Let's create one in AppConfig and Check.

Reply:
ok, i have already edited it.